qmxgraph/api.py

```diff
@@ -339,6 +339,38 @@ def is_visible(self, cell_id):
         """
         return self.call_api('isVisible', cell_id)
 
+    def zoom_in(self):
+        """
+        Zoom in the graph.
+        """
+        return self.call_api('zoomIn')
+
+    def zoom_out(self):
+        """
+        Zoom out the graph.
+        """
+        return self.call_api('zoomOut')
+
+    def reset_zoom(self):
+        """
+        Reset graph's zoom.
+        """
+        return self.call_api('resetZoom')
+
+    def fit(self):
+        """
+        Rescale the graph to fit in the container.
+        """
+        return self.call_api('fit')
+
+    def get_zoom_scale(self):
```

Comment:
Will we be able to do any extra work to save the final scene in the project file, with pans, zooms and whatever changes the user made?

Reply:
I went to actually check that since I was expecting it to already be saved on `dump` but it is not.
It should be easy since we can obtain all that data.

Reply:
:+1:

Reply:
Added `get_scale_and_translation`/`set_scale_and_translation` to abstract the order pan and zoom are applied.